packages/clarity-js/src/interaction/pointer.ts

```diff
@@ -129,7 +129,8 @@ function similar(last: PointerState, current: PointerState): boolean {
     let distance = Math.sqrt(dx * dx + dy * dy);
     let gap = current.time - last.time;
     let match = current.data.target === last.data.target;
-    return current.event === last.event && match && distance < Setting.Distance && gap < Setting.Interval;
+    let sameId = current.data.id !== undefined ? current.data.id === last.data.id : true;
```

Comment:
this could be tricky, the same element can have multiple ids, think a searchbox with a div inside set up for margin. do you know how much more data we get with this additional check ?

Reply:
This isn't element id. This is touch id to differentiate between primary vs secondary touches